docs/INSTALLING.md

```diff
@@ -0,0 +1,71 @@
+# How do I install the Code Corps Ember app?
+
+### Requirements
+
+You will need to install the following: 
+
+- [Node.js/npm](https://docs.npmjs.com/getting-started/installing-node) we use the LTS version (v 4.4.7)
+- bower: `npm install -g bower`
+- ember-cli: `npm install -g ember-cli`
```

Comment:
I beleive this covers the bases, however if anyone notices something missing let me know